specifyweb/frontend/js_src/lib/tests/fixtures/uploadplan.1.json

```diff
@@ -283,34 +283,40 @@
                             "default": null,
                             "column": "Class"
                           }
-                        }
+                        },
+                        "treeId": 1
                       },
                       "Family": {
                         "treeNodeCols": {
                           "name": "Family"
-                        }
+                        },
+                        "treeId": 1
```

Comment:
Why do I care about this? You cannot, ever, select ranks from different trees, correct? Like, say, Phylum from tree1 and Species from tree 4? You are overcomplicating design, making things easier to break, for something that just isn't needed. So, get rid of treeId from here, and just make it at the top for treerecord.

Reply:
@acwhite211 do the adjustments on the backend, and see if it can be simplified.

Reply:
The front-end's design allowed for selecting ranks from multiple trees, so the back-end got adjusted to handle it.  It would have been better probably to just use the unique db id for Treedefid and Treedefitemid to identify ranks throughout Specify, rather than by name.